package.yaml

```diff
@@ -47,8 +47,6 @@ ghc-options:
   - -fno-sort-valid-hole-fits
   - -optl=-lLLVM-14
 
-system-build-tools: llvm-config
```

Comment:
this gets rid of these warnings during build:
```
Warning:
~/placidex/llvm-codegen/llvm-codegen.cabal:108:3: The
field "build-tools" is deprecated in the Cabal specification version 2.0.
Please use 'build-tool-depends' field
Warning:
~/placidex/llvm-codegen/llvm-codegen.cabal:64:3: The
field "build-tools" is deprecated in the Cabal specification version 2.0.
Please use 'build-tool-depends' field
```

Reply:
No we actually need this, we need llvm14 to fully use the package. This checks for that. The real fix is to ditch hpack and move to cabal-fmt (maybe in another PR)

Reply:
reverted the change to `package.yaml`